include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
+    /* VK_AMD_programmable_sample_locations */
+    /* VK_AMD_rasterization_order */
+    /* VK_AMD_shader_ballot */
+    /* VK_AMD_shader_explicit_vertex_parameter */
+    /* VK_AMD_shader_fragment_mask */
+    /* VK_AMD_shader_image_load_store_lod */
+    /* VK_AMD_shader_info */
+    /* VK_AMD_shader_trinary_minmax */
+    /* VK_AMD_texture_gather_bias_lod */
+    /* VK_AMD_wave_limits */
+    /* VK_ANDROID_external_memory_android_hardware_buffer */
+    /* VK_ANDROID_native_buffer */
+    /* VK_EXT_2d_imageview_3d_image */
+    /* VK_EXT_calibrated_timestamps */
+    /* VK_EXT_debug_marker */
+    /* VK_EXT_debug_report */
+    /* VK_EXT_debug_utils */
+    /* VK_EXT_depth_clip_control */
+    /* VK_EXT_depth_clip_enable */
+    /* VK_EXT_depth_range_unrestricted */
+    /* VK_EXT_device_memory_report */
+    /* VK_EXT_display_control */
+    /* VK_EXT_external_memory_dma_buf */
+    /* VK_EXT_filter_cubic */
+    /* VK_EXT_full_screen_exclusive */
+    /* VK_EXT_global_priority_query */
+    /* VK_EXT_hdr_metadata */
+    /* VK_EXT_image_drm_format_modifier */
+    /* VK_EXT_load_store_op_none */
+    /* VK_EXT_memory_priority */
+    /* VK_EXT_metal_objects */
+    /* VK_EXT_non_seamless_cube_map */
+    /* VK_EXT_physical_device_drm */
+    /* VK_EXT_pipeline_creation_feedback */
+    /* VK_EXT_post_depth_coverage */
+    /* VK_EXT_queue_family_foreign */
+    /* VK_EXT_rgba10x6_formats */
+    /* VK_EXT_separate_stencil_usage */
+    /* VK_EXT_shader_module_identifier */
+    /* VK_EXT_shader_stencil_export */
+    /* VK_EXT_swapchain_colorspace */
+    /* VK_EXT_tooling_info */
+    /* VK_EXT_validation_cache */
+    /* VK_EXT_vertex_input_dynamic_state */
+    /* VK_EXT_video_decode_h264 */
+    /* VK_EXT_video_decode_h265 */
+    /* VK_EXT_video_encode_h264 */
+    /* VK_EXTX_portability_subset */
+    /* VK_GOOGLE_decorate_string */
+    /* VK_GOOGLE_display_timing */
+    /* VK_GOOGLE_hlsl_functionality1 */
+    /* VK_GOOGLE_sampler_filtering_precision */
+    /* VK_GOOGLE_user_type */
+    /* VK_HUAWEI_prerotation */
+    /* VK_HUAWEI_smart_cache */
+    /* VK_IMG_filter_cubic */
+
     // [DO NOT EXPOSE] Enables certain formats in Vulkan, we just enable them if available or else we need to make format support query functions in LogicalDevice as well
     /* VK_IMG_format_pvrtc */
-   
+
+    /* VK_INTEL_performance_query */
+    /* VK_KHR_bind_memory2 */
+    /* VK_KHR_copy_commands2 */
+    /* VK_KHR_create_renderpass2 */
+    /* VK_KHR_deferred_host_operations */
+    /* VK_KHR_descriptor_update_template */
+    /* VK_KHR_device_group */
+    /* VK_KHR_display_swapchain */
+    /* VK_KHR_external_fence */
+    /* VK_KHR_external_fence_fd */
+    /* VK_KHR_external_fence_win32 */
+    /* VK_KHR_external_memory */
+    /* VK_KHR_external_memory_fd */
+    /* VK_KHR_external_memory_win32 */
+    /* VK_KHR_external_semaphore */
+    /* VK_KHR_external_semaphore_fd */
+    /* VK_KHR_external_semaphore_win32 */
```

Comment:
same as with `external_memory` promoted to 1.1 core, always enable, then report relevant things as limit